lib/logstash/filters/aggregate.rb

```diff
@@ -195,6 +195,30 @@ class LogStash::Filters::Aggregate < LogStash::Filters::Base
   # Example value : `"map['sql_duration'] += event['duration']"`
   config :code, :validate => :string, :required => true
 
+
+
+  # The code to execute to complete timeout generated event, when 'push_map_as_event_on_timeout' or 'push_previous_map_as_event' is set to true. 
+  # The code block will have access to the newly generated timeout event that is pre-populated with the aggregation_map values. 
```

Comment:
Could you just replace "with the aggregation_map values" by "with aggregation map".
By the way, it allows to coherent this "push_map_as_event_on_timeout" description.